styles.css

```diff
@@ -0,0 +1,789 @@
+  margin-bottom: 30px;
+}
+
+.languages-2 li:nth-child(n) {
+  border: 1px;
+  border-radius: 8px;
+  padding: 5px 12px 5px 12px;
+  background-color: #ebebff;
+  color: #6070ff;
+  letter-spacing: 3%;
+  font-size: 15px;
+  text-emphasis-style: medium;
+  text-align: center;
+  margin-top: 30px;
+  list-style: none;
+}
+
+.w-title ul li:nth-child(n + 2) {
+  color: #6b778c;
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+  line-height: 24px;
+  font-weight: 500;
+  list-style-image: url(./images/dot1.png);
+  margin-left: 20px;
+}
+
+.p-text {
+  font-size: 15px;
+  font-weight: 400;
+  font-style: "Poppins", sans-serif;
+  color: #344563;
+  vertical-align: text-top;
+  line-height: 24px;
+  margin: 0;
+}
+
+.canop {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  align-items: baseline;
+  align-self: flex-start;
+  gap: 10px;
+  list-style: none;
+}
+
+.tap {
+  cursor: pointer;
+  padding: 15px;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 700;
+  background-color: #fff;
+  border-radius: 8px;
+  border: 2px solid #396df2;
+}
+
+/* media query for the above card */
+@media (max-width: 470px) {
+  .middle-section {
+    align-items: flex-start;
+    flex-direction: column;
+  }
+
+  .card1 {
+    display: block;
+    width: 100%;
+    background-size: contain;
+    height: 80%;
+    margin-left: 10px;
+  }
+
+  .responsive {
+    width: 15%;
+    height: 25%;
+    margin-left: 40px;
+    margin-right: auto;
+  }
+
+  .name {
+    font-size: 20px;
+    margin-left: 20px;
+  }
+
+  .languages li:nth-child(n) {
+    font-size: 10px;
+    margin-left: 20px;
+  }
+
+  .w-title ul li:nth-child(n + 2) {
+    font-size: 9px;
+  }
+
+  .word {
+    width: 20%;
+    font-size: 10px;
+    margin-left: 20px;
+  }
+
+  .languages {
+    padding-bottom: 0;
+  }
+
+  .tap {
+    width: 10%;
+    height: 3%;
+    font-size: 10px;
+  }
+
+  button.tap {
+    position: relative;
+    margin-left: 80px;
+  }
+}
+
+/* end of media quary for this session */
+
+.card2 {
+  background-color: #fff;
+  border: 1px white;
+  border-radius: 24px;
+  padding-left: 5px;
+  height: 496px;
+  width: 1156px;
+  margin-top: 0;
+  display: grid;
+  margin-bottom: 60px;
+  grid-auto-flow: column;
+}
+
+.snapt {
+  order: 1;
+  max-height: 450px;
+  max-width: 545px;
+}
+
+.snapt img {
+  height: 448px;
+  width: 544px;
+  padding-right: 40px;
+  padding-top: 20px;
+}
+
+.titol {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  padding-left: 48px;
+}
+
+.snapshooti img {
+  height: 430px;
+  width: 500px;
+  margin: 40px 20px 0 20px;
+}
+
+.languages-2 {
+  display: flex;
+  align-self: flex-start;
+  padding-left: 0;
+  gap: 10px;
+  margin-top: 15px;
+}
+
+.blind {
+  cursor: pointer;
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 700;
+  background-color: #fff;
+  border-radius: 8px;
+  border: 2px solid #396df2;
+  max-width: 120px;
+  max-height: 110px;
+  padding: 10px;
+  margin-top: 10px;
+}
+
+.see {
```

Comment:
- [ ] As the previous code reviewer suggested. I also kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-26 23-47-31](https://user-images.githubusercontent.com/52934128/165405182-05da2572-b77c-4e42-88a6-9f950471af8d.png)

**the button on the pressed state has the `background: #2230D2;`**

### EXAMPLE:
```
button:active {
   background-color: #2230D2;
}
```
**the button on the pressed state has the `background: #6070FF;`**

```
button:hover {
    background-color: #6070FF;
}
```